src/Http/Middleware/HandleErrors.php

```diff
@@ -52,7 +52,7 @@ public function __invoke($error, Request $request, Response $response, callable
             $status = $errorCode;
         }
 
-        if ($this->debug && $errorCode !== 404) {
+        if ($this->debug && ! in_array($errorCode, [403, 404])) {
```

Comment:
To me, this behavior is a feature. When somebody tries to access a page that they are not allowed to see, they should get a pretty and helpful error page saying so. 

In debug mode, you want to quickly find information about unexpected errors. This one is not unexpected. We are just using exceptions as an implementation detail. 

If a developer needs a more detailed stack trace, they can always throw a random exception...